tests/baseclass.py

```diff
@@ -46,6 +47,25 @@ def setUp(self):
         self._device = _ped.device_get(self.path)
         self.device = parted.getDevice(self.path)
 
+# Base class for any test case, which requires loop block device - need run
+# as root
+class RequiresLoopDisk(unittest.TestCase):
+    def setUp(self):
+        tmp_file = "/tmp/temp_loop"
+        f = subprocess.Popen("fallocate -l 15M " + tmp_file, shell=True)
+        f.communicate()
+        l = subprocess.Popen("losetup -f " + tmp_file + " --show", shell=True, stdout=subprocess.PIPE)
+        self.path = l.communicate()[0].strip()
+
+        self._device = _ped.device_get(self.path)
+        self.device = parted.getDevice(self.path)
+        self._disk = _ped.disk_new_fresh(self._device, _ped.disk_type_get("msdos"))
+        self.disk = parted.Disk(PedDisk=self._disk)
+
+        if self.path and os.path.exists(self.path):
```

Comment:
Hmmm, that looks like it needs its own method. At least the other classes have it.